sims/vcs/Makefile

```diff
@@ -50,7 +50,8 @@ VCS_CC_OPTS = \
 	-CC "-I$(dramsim_dir)" \
 	-CC "-std=c++11" \
 	$(dramsim_lib) \
-	$(RISCV)/lib/libfesvr.a
+	$(RISCV)/lib/libfesvr.a \
+	-CC "$(EXTRA_SIM_CC_FLAGS)"
```

Comment:
Does this work if there are more than 1 extra flags?

Reply:
Yup. If there are extra flags in the `EXTRA_SIM_CC_FLAGS` this works.

Reply:
Are you sure that vcs does the right thing w.r.t. `-CC "-Iblah -Iblah2"`

Reply:
I just tried the `-CC "-I$(VCS_HOME)/include -I$(RISCV)/include -I$(dramsim_dir) -std=c++11"` on one line and that worked.